src/main/java/org/opentripplanner/netex/mapping/StopAndStationMapper.java

```diff
@@ -129,9 +130,9 @@ private Collection<FareZone> mapTariffZones(StopPlace stopPlace) {
 
     return stopPlace
       .getTariffZones()
-      .getTariffZoneRef()
+      .getTariffZoneRef_()
```

Comment:
Out of curiosity: what do the underscores mean?

Reply:
The naming conventions are a bit obscure but it seems that it means that the method return wrapped elements (JAXBElement) rather than the element itself